### PR TITLE
doc: add description of metric fields for cephfs-top

### PR DESCRIPTION
**Pull request by neesingh-rh**
Fixes: https://tracker.ceph.com/issues/63991
Signed-off-by: Neeraj Pratap Singh <neesingh@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/cephfs/cephfs-top.rst
Comment (anthonyeleven) [inline doc/cephfs/cephfs-top.rst:76]: Suggest removing "It is" from all of these.

Comment (zdover23) [reply doc/cephfs/cephfs-top.rst:76]: I'll make this edit. 

@neesingh-rh, You don't have to do anything. I'll make sure this gets altered, merged, and backported. Thanks for the contribution!

Comment (neesingh-rh) [reply doc/cephfs/cephfs-top.rst]: > I'll make this edit.
> 
> @neesingh-rh, You don't have to do anything. I'll make sure this gets altered, merged, and backported. Thanks for the contribution!

I already did push the changes addressing above suggestion. Beyond this, you can take care of this. And pls don't merge until @joscollin or @vshankar approves from @ceph/cephfs . Thanks.

**Comment by zdover23** [comment]
jenkins retest this please

File: doc/cephfs/cephfs-top.rst
Comment (joscollin) [inline doc/cephfs/cephfs-top.rst:106]: image

Can't differentiate the `abbreviation`, `expansion phrase` and the `description`. Could you please bring a visual difference between these (for all fields)?

**Review by joscollin — CHANGES_REQUESTED** [review]

File: doc/cephfs/cephfs-top.rst
Comment (vshankar) [inline doc/cephfs/cephfs-top.rst:70]: Percentage of file capability hits over total number of caps.

File: doc/cephfs/cephfs-top.rst
Comment (vshankar) [inline doc/cephfs/cephfs-top.rst:97]: What does "last refresh" mean?

**Review by vshankar — CHANGES_REQUESTED** [review]

File: doc/cephfs/cephfs-top.rst
Comment (neesingh-rh) [reply doc/cephfs/cephfs-top.rst:97]: I took it from https://github.com/ceph/ceph/commit/ec22208364086605361c2f7f54f6b9908fa8bc7e#diff-c739b905b3b881e50bd7523f0f5c602709f7b7bf30881d0e17b07e5c9524741cR84 . I suppose it means with respect to the duration from the last `refresh_client`.

File: doc/cephfs/cephfs-top.rst
Comment (neesingh-rh) [reply doc/cephfs/cephfs-top.rst]: PTAL

Comment (joscollin) [reply doc/cephfs/cephfs-top.rst]: Do the same for description too. Something like https://ceph--55465.org.readthedocs.build/en/55465/cephfs/cephfs-top/#interactive-commands would be readable.

File: doc/cephfs/cephfs-top.rst
Comment (vshankar) [reply doc/cephfs/cephfs-top.rst]: yeh, that's a code comment which is meant to refer code blocks. We should be _much_ clear in the docs.

Comment (neesingh-rh) [reply doc/cephfs/cephfs-top.rst]: How about now, PTAL?

File: doc/cephfs/cephfs-top.rst
Comment (neesingh-rh) [reply doc/cephfs/cephfs-top.rst]: I did the same way it is done for `interactive commands`, pls have  a look.

Comment (joscollin) [reply doc/cephfs/cephfs-top.rst]: Please see the build https://ceph--55465.org.readthedocs.build/en/55465/cephfs/cephfs-top/#description-of-fields

**Review by joscollin — APPROVED** [review]

**Review by vshankar — APPROVED** [review]
Nice work @neesingh-rh

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55511 - Reef backport
https://github.com/ceph/ceph/pull/55512 - Quincy backport

**Pull request merged**
